src/olympia/versions/utils.py

```diff
@@ -52,12 +52,15 @@ def write_svg_to_png(svg_content, out):
         return convert_svg_to_png(temporary_svg.name, out)
 
 
+SVG_DIMENSIONS_REGEX = rb'(?=.* width="(?P<width>\d+)")(?=.* height="(?P<height>\d+)")'
```

Comment:
What would happen if the SVG is declared like this:

```
<svg height="92" width="720" ...>
```

(width and height attrs inverted)

Reply:
added some tests https://github.com/mozilla/addons-server/pull/17859/commits/f395f45f48fb21d6a85baa2971c3360097dce3f5